src/Styles/MenuBar.less

```diff
@@ -3,17 +3,24 @@
     right: 25px;
     top: 20px;
     height: 30px;
+    border: 1px solid rgba(255,255,255,0.1);
```

Comment:
It's a fairly subtle difference, but I'd consider it noticeable:
 
![screen shot 2015-01-21 at 4 25 30 pm](https://cloud.githubusercontent.com/assets/5292858/5831636/91491be8-a18a-11e4-9cd0-8adbe4c8f7c7.png)
![screen shot 2015-01-21 at 4 27 16 pm](https://cloud.githubusercontent.com/assets/5292858/5831639/9a07d1ca-a18a-11e4-9063-7b50d5029fe9.png)

The border itself isn't essential – that said, i'm inclined to keep it matching the internal menu bar dividers while it's there.